spec/did_you_know_spec.rb

```diff
@@ -0,0 +1,68 @@
+RSpec.describe Geordi::Hint do
+
+  subject { Geordi::Hint }
+
+  describe '#did_you_know' do
+
+    it 'prints a random hint in the console if an array with suitable suggestions was provided and returns all generated hints' do
```

Comment:
Since the return value is not relevant for the "Did you know" feature, it should not be mentioned in test descriptions. Also, the condition is not needed in the example description – it is a case that does not occur in the example. Suggested description:

```suggestion
    it 'generates hints from its arguments and prints one of them to stdout' do
```